동현/11주차/11725.py

```diff
@@ -0,0 +1,24 @@
+# 11725 트리의 부모찾기
+
+import sys
+sys.setrecursionlimit(10**6)        # 최대 깊이 제한
+n = int(sys.stdin.readline()) 
+
+graph = [[] for _ in range(n+1)]    # graph 선언
+for i in range(n-1):
+    point1, point2 = map(int, sys.stdin.readline().split()) # 입력값 저장
+    graph[point1].append(point2)    # 이차원 배열 저장
+    graph[point2].append(point1)
+
+visited = [0] * (n+1)               # 방문처리 기록할 visited list 초기화
+
+def dfs(s):                         # dfs
```

Comment:
단순 변수명을 s라고 하니까 한눈에 알아보기가 힘드네요
start로 변경해주시는게 조금 더 가독성이 좋을듯합니다!
```suggestion
def dfs(start):                         # dfs
```